docs/reference/signals.rst

```diff
@@ -41,9 +41,8 @@ example showing how you might notify your team when something is published:
             "icon_emoji": ":octopus:"
         }
 
-        data = urllib.urlencode(values)
-        req = urllib2.Request(url, data)
-        response = urllib2.urlopen(req)
+        data = json.dumps(values)
```

Comment:
A small note, the `json.dumps` step is redundant as `requests.post` accepts json serializable datatypes (such as dict). See this example: https://requests.readthedocs.io/en/master/user/quickstart/#more-complicated-post-requests

Reply:
thanks for the suggestion @marteinn! I learned something new today. I'll make the change right now and resubmit